packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -19,15 +19,14 @@ type RovingFocusGroupOptions = {
 type RovingContextValue = {
   groupId: string;
   tabStopId: string | null;
-  setTabStopId: React.Dispatch<React.SetStateAction<string | null>>;
   reachable: boolean;
-  setReachable: React.Dispatch<React.SetStateAction<boolean | undefined>>;
+  onTabStopIdChange(itemId: string, active?: boolean): void;
```

Comment:
tiny nit but I think it's nice to group the prop and change handlers together

Reply:
Since you allowed my `onFoo(): void` changes from `onFoo: () => void` I'll happily oblige here 😉  I usually group all my handlers at the end but happy to see how the alternative feels

Reply:
haha, I think the other places were left grouped also no?

Reply:
coincidentally grouped perhaps haha, i tend not to look for it